Assets/BossRoom/Prefabs/BossRoomHudCanvas.prefab.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
For this "HowToPlayPopupPanel" prefab I like that the popup panel prefab reference was broken but the PopupPanel script still remains. That can safely go.

Reply:
ok